src/Writer/Extension/PodcastIndex/Renderer/Entry.php

```diff
@@ -54,20 +81,15 @@ protected function _appendNamespaces(): void
      */
     protected function setTranscript(DOMDocument $dom, DOMElement $root): void
     {
-        /** @psalm-var null|array<string, string> $locked */
-        $locked = $this->getDataContainer()->getPodcastIndexTranscript();
-        if ($locked === null) {
+        /** @psalm-var EntryWriter $container */
+        $container = $this->getDataContainer();
+
+        /** @psalm-var null|TranscriptArray $transcript */
```

Comment:
Forcing the var here should be superfluous - it should already be in the return type of `getPodcastIndexTranscript`